### PR TITLE
bootloader_setup: Fix count in add_custom_grub_entries()

### PR DESCRIPTION
**Pull request by pevik**
It's problematic for cases where are more than one kernel installed
https://openqa.suse.de/tests/1812945#step/install_ltp/53

- Verification run:
http://quasar.suse.cz/tests/594
http://quasar.suse.cz/tests/595
http://quasar.suse.cz/tests/596
http://quasar.suse.cz/tests/597

### CODE REVIEW

**Comment by pevik** [comment]
@mimi1vx @ostruzina FYI

File: lib/bootloader_setup.pm [hunk 1]
```diff
@@ -84,30 +84,39 @@ And of course the new entries have C<ima_policy=tcb> added to kernel parameters.
 sub add_custom_grub_entries {
     my $grub_param = get_var('GRUB_PARAM');
     return unless defined($grub_param);
+    my $script_old     = "/etc/grub.d/10_linux";
```
Comment (asmorodskyi) [inline lib/bootloader_setup.pm]: in a fact it is menuentry, 'script' in a variable name looks confusing for me

**Comment by mimi1vx** [comment]
@richiejp please merge

File: lib/bootloader_setup.pm (hunk 1 above)
Comment (pevik) [reply lib/bootloader_setup.pm]: @asmorodskyi If you look inside, it's a shell script.

**Pull request merged**